apps/fhir/bluebutton/models.py

```diff
@@ -24,6 +24,17 @@ def get_queryset(self):
         return super().get_queryset().filter(Q(fhir_id__startswith='-'))
 
 
+def hash_hicn(hicn):
```

Comment:
Validate hicn format.

Test cases:
- bad format
- empty string
- happy case

Reply:
Is this still TBD based on confirmation of what HICN format is supposed to look like?

Reply:
yeah, even if it doesn't make it in this PR it's at least documented.

Reply:
I've added a very basic not blank check. This will at least start the pattern in preparation for stronger validation.